### PR TITLE
Add general tune-ups, correct typos, and add articles

### PR DESCRIPTION
**Pull request by adswa**

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -42,65 +42,69 @@ Born of idea to provide a unified data distribution for neuroscience, taking a v
...
+The DataLad project's initial goal was to provide a data distribution with unified access to the already available public data archives in neuroscience, such as http://crcns.org or http://openfmri.org.
+Git and git-annex are just "client tools" and do not provide aggregation of different resources into a unified distribution on their own.
+http://datasets.datalad.org became an example of such a data distribution curated by the DataLad team, which at the moment provides streamlined access to over 250 TBs of data across wide range of projects and archives.
 
 **Modularization is needed to scale.**
 Research workflows impose additional demands for an efficient research data management (RDM) platform besides "version control" and "data transport".
-Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single git repository even if files are tiny in their size.
+Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single git repository even if individual files are tiny in their size.
```
Comment (bpoldrack) [inline paper.md:76]: Not sure, where to put this comment, so abusing your PR ;-)

I think there's another aspect of modularization, that we didn't mention yet, but relates closely to the previously pointed to notion of "like software distribution". That is the notion of dependencies. Not sure whether to point that out in this paragraph or rather early on, when we mention the original idea of "data packages like software packages".

Comment (mih) [reply paper.md:76]: Better to put a comment in the manuscript. It will get lost here.

File: paper.md [hunk 2]
```diff
@@ -42,65 +42,69 @@ Born of idea to provide a unified data distribution for neuroscience, taking a v
...
 Disconnected FAIR data hosting portals provide a cacophony of data access and authentication methods, data versioning is frequently ignored, and shared data provenance is often not recoverable simply because data management is rarely considered to be an integral part of the scientific process.
+DataLad aims to solve these issues by streamlining data consumption, publication, and updating routines, by providing a simplified core API for git and git-annex operations, and by providing additional features for reproducible science such as command execution and automatic recomputation based on complete process provenance.
+As it is interoperable with a large variety of scientific and commercial hosting services and available for all operating systems, DataLad can be integrated into established systems with minimal adjustments.
 
 ## Why git and git-annex
 
-Git is an excellent distributed content management system geared toward management and collaboration on text files, and as such cannot handle large or binary files well.
```
Comment (mih) [inline paper.md:48]: > Git is an excellent distributed content management system

Probably best to stick to general phrasing in scientific papers. Unless there is a citation or a solid argument why it is excellent, this is not the place to make such claims IMHO

File: paper.md [hunk 3]
```diff
@@ -42,65 +42,69 @@ Born of idea to provide a unified data distribution for neuroscience, taking a v
...
 
-`datalad` Python package provides both a Python library and a command line tool which expose core DataLad functionality to fulfill a wide range of dRDM use cases for any field of endeavor (see Figure 1).
+The `datalad` Python package provides both a Python library and a command line tool which expose core DataLad functionality to fulfill a wide range of dRDM use cases for any domain.
 Its API (see \autoref{fig:one}) operates on DataLad datasets which are just git (with optional git-annex for data) repositories with additional metadata and configuration.
  
 ![DataLad: overview of available commands for various parts of the data management process \label{fig:one}](figures/datalad_process.png)
 
-Based on built-in mechanism of git submodules, DataLad embraces and simplifies modular composition of smaller datasets into a larger (super)datasets.
-With this simple paradigm, DataLad fulfills YODA principles [@yoda:myyoda] and facilitates efficient access, composition, scalability, reuse, sharing, and reproducibility of results  (see Figure 2).
+Based git's submodules mechanism, DataLad embraces and simplifies modular composition of smaller datasets into larger (super)datasets.
```
Comment (yarikoptic) [inline paper.md]:
```suggestion
Using git's submodules mechanism, DataLad embraces and simplifies modular composition of smaller datasets into larger (super)datasets.
```

File: paper.md [hunk 4]
```diff
@@ -124,25 +129,25 @@ https://www.datalad.org/integrations.html provides a more complete list of DataL
...
 Any reported dataset could be immediately installed using `datalad install` command, and data files of interest obtained using `datalad get`.
 
 ## Installation
 
-DataLad Handbook provides [installation instructions](http://handbook.datalad.org/en/latest/intro/installation.html) for all operating systems.
+The DataLad Handbook provides [installation instructions](http://handbook.datalad.org/en/latest/intro/installation.html) for all operating systems.
 DataLad releases are distributed through PyPI, Debian, NeuroDebian, brew, conda-forge.
-[datalad-installer](https://github.com/datalad/datalad-installer/) (also available from PyPI) can be used to streamline installation of `git-annex` which is a dependency of the DataLad, and thus cannot be installed via `pip`. 
+The [datalad-installer](https://github.com/datalad/datalad-installer/) (also available from PyPI) can be used to streamline the installation of `git-annex`, which cannot be installed via `pip` and thus may need a separate installation, across operating systems.
```
Comment (yarikoptic) [inline paper.md]:
```suggestion
The [datalad-installer](https://github.com/datalad/datalad-installer/) (also available from PyPI) can be used to streamline the installation of `git-annex`, which cannot be installed via `pip` and thus may need a separate installation on some operating systems.
```

**Comment by yarikoptic** [comment]
Thank you @adswa !

Since conflicts are tricky with text, I quickly tuned up just a bit and merged so @bpoldrack could add a PR with an additional aspect

**Pull request merged**